packages/TypeDeclaration/tests/Rector/Property/PropertyTypeDeclarationRector/Fixture/doctrine_relations.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+namespace Rector\TypeDeclaration\Tests\Rector\FunctionLike\PropertyTypeDeclarationRector\Fixture {
+    class Product
+    {
+    }
+
+    class Car
+    {
+    }
+}
+
+namespace Rector\TypeDeclaration\Tests\Rector\FunctionLike\PropertyTypeDeclarationRector\Fixture\Entity {
+    class Company
+    {
+    }
+}
```

Comment:
These should be in `/Source`.

`/Fixture` directory is only for tested code (a code that will be changed)